src/components/PodmanStatus.jsx

```diff
@@ -60,7 +60,7 @@ export default class PodmanStatus extends React.Component {
                         <td>
                             <Progress value={this.state.DiskUse}
                                 className="pf-m-sm"
-                                min={0} max={Number(this.state.DiskSize)}
+                                min={0} max={ this.state.DiskSize == 0 ? Number(1) : Number(this.state.DiskSize) }
```

Comment:
why make it `Number(1)` and not `Number(0)

Reply:
https://github.com/patternfly/patternfly/issues/4582